requirements.txt

```diff
@@ -25,7 +25,7 @@ flask==2.1.3              # via reana-job-controller (setup.py)
 fs==2.4.16                # via reana-commons, reana-job-controller (setup.py)
 google-auth==2.16.2       # via kubernetes
 gssapi==1.8.2             # via paramiko
-htcondor==8.9.11          # via reana-job-controller (setup.py)
+htcondor==9.0.0           # via reana-job-controller (setup.py)
```

Comment:
We can use `9.0.17` as on LXPLUS8. I have verified that this is also the latest Focal version installed by upstream HTCondor binary package repository:

```
root@534752f5271d:/code# dpkg -l | grep htcondor
ii  htcondor                           9.0.17-3.1                        amd64        distributed workload management system
```